app/models/todo.rb

```diff
@@ -1,2 +1,5 @@
 class Todo < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 25 }
+  validates :status, presence: true
+  validates :is_completed, inclusion: [true, false]
```

Comment:
I know this looks odd but `presence: true` will always fails on any falsy value. Ruby treats `false` and `"false"` as falsy. This is the best way to check for presence of a boolean value.